google-cloud-bigquery/lib/google-cloud-bigquery.rb

```diff
@@ -43,6 +43,8 @@ module Cloud
     # @param [Integer] retries Number of times to retry requests on server
     #   error. The default value is `5`. Optional.
     # @param [Integer] timeout Default request timeout in seconds. Optional.
+    # @param [String] endpoint Override of the endpoint host name. Optional.
```

Comment:
We didn't add `endpoint` to `Google::Cloud.pubsub` or `Google::Cloud#pubsub`, only to `Google::Cloud::PubSub.new`. The `Google::Cloud` methods are not labeled as deprecated, but we aren't encouraging their use. I would prefer we don't update them.

Reply:
SGTM. Removed.